src/pkg/cli/tail.go

```diff
@@ -65,12 +65,14 @@ type TailOptions struct {
 type P = client.Property // shorthand for tracking properties
 
 func CreateEndLogEventDetectFunc(conditionals []EndLogConditional) TailDetectStopEventFunc {
-	return func(service string, host string, eventLog string) bool {
+	return func(services []string, host string, eventLog string) bool {
 		for _, conditional := range conditionals {
-			if service == "" || service == conditional.Service {
-				if host == "" || host == conditional.Host {
-					if strings.Contains(eventLog, conditional.EventLog) {
-						return true
+			for _, service := range services {
```

Comment:
Do we still need this? For BYOC maybe? If so, maybe we should move this code into the BYOC client instead?

Reply:
Will need this for playground until we implement server side mapping of cd logs to service state.